regparser/default_settings.py

```diff
@@ -72,6 +72,9 @@
     'regcontent.overrides'
 ]
 
+# list of fr notice overrides
+FR_NOTICE_OVERRIDES = {'ALL': []}
```

Comment:
This will probably change when merge conflicts are resolved, but it should be just a plain dict (`{}`).